components/nvs_flash/include/nvs_flash.h

```diff
@@ -61,6 +61,19 @@ esp_err_t nvs_flash_init(void);
  */
 esp_err_t nvs_flash_init_partition(const char *partition_label);
 
+/**
+ * @brief Initialize NVS flash storage for the specified custom partition.
+ *
+ * @param[in]   Pointer to somehow obtained partition.
+ *
+ * @return
+ *      - ESP_OK if storage was successfully initialized.
+ *      - ESP_ERR_NVS_NO_FREE_PAGES if the NVS storage contains no empty pages
```

Comment:
Please adjust the return value documentation to match the code (I know it's obvious, but please mention the `ESP_ERR_INVALID_ARG` in case partition or partition->label in NULL).

Reply:
Fixed